datafusion/core/tests/parquet/arrow_statistics.rs

```diff
@@ -592,58 +739,180 @@ async fn test_timestamp_diff_rg_sizes() {
     }
     .run();
 
+    Test {
+        reader: reader.build().await,
+        // mins are [2020-01-01T12:01:01+11:00, 2020-01-01T12:11:01+11:00, 2020-01-11T12:02:01+11:00]
+        expected_min: Arc::new(
+            TimestampNanosecondArray::from(vec![
+                TimestampNanosecondType::parse("2020-01-01T01:01:01"),
+                TimestampNanosecondType::parse("2020-01-01T01:11:01"),
+                TimestampNanosecondType::parse("2020-01-11T01:02:01"),
+            ])
+            .with_timezone(tz),
+        ),
+        // maxes are [2020-01-02T01:01:01+11:00, 2020-01-11T01:01:01+11:00, 2020-01-12T01:01:01+11:00]
+        expected_max: Arc::new(
+            TimestampNanosecondArray::from(vec![
+                TimestampNanosecondType::parse("2020-01-02T01:01:01"),
+                TimestampNanosecondType::parse("2020-01-11T01:01:01"),
+                TimestampNanosecondType::parse("2020-01-12T01:01:01"),
+            ])
+            .with_timezone(tz),
+        ),
+        // nulls are [1, 2, 1]
+        expected_null_counts: UInt64Array::from(vec![1, 2, 1]),
+        // row counts are [8, 8, 4]
+        expected_row_counts: UInt64Array::from(vec![8, 8, 4]),
+        column_name: "nanos_timezoned",
+    }
+    .run();
+
     // micros
     Test {
         reader: reader.build().await,
-        expected_min: Arc::new(Int64Array::from(vec![
-            1577840461000000,
-            1577841061000000,
-            1578704521000000,
+        // mins are [2020-01-01T01:01:01, 2020-01-01T01:11:01, 2020-01-11T01:02:01]
```

Comment:
these comments seem a little redundant given that the actual timestamp string appears in the test as well

Reply:
Done.